content/docs/platform/workflow/translations.mdx

```diff
@@ -0,0 +1,133 @@
+- Customize language per workflow or layout.
+- Reduce friction in communication across geographies.
+
+Translations can be managed on the [translation page](https://dashboard.novu.co/translations) on Novu dashboard. Select the default language and add the target languages you wish to support.
+
+![Translation page](/images/platform/workflow/translations/translations-page.png)
+
+## Enabling translations
+
+1. **While creating a new workflow**
+
+- Check the **Enable Translations** toggle.
+- Set the default locale (e.g., `en_US`).
+- Add target locales you wish to support (e.g., `fr_FR`, `es_ES`, `ja_JP`, etc.).
+
+2. **While editing an existing workflow**
+
+- Open an existing workflow.
+- Click the **Enable Translations** option in the right panel
+- Enable the toggle if not already enabled.
+
+Default and targeted locales can be configured on the translation page.
+
+![Translation locale](/images/platform/workflow/translations/translations-default-and-target-locale.gif)
+
+## Translation Groups
+A translation group contains all language variants for a workflow identified by a unique workflow ID.
+
+You can:
+
+- Search for groups on the **Translations** page.
+- View status per locale ('Outdated' or 'Up to date').
+- Import or export translations for a specific locale (e.g., `en_US`).
+- Import or export translations for all workflows where translations are enabled.
+
+## Translation JSON format
+
+Each locale supports JSON-based translation files. For example, `en_US.json` file contains the default translations for the workflow. Translation JSON files can be updated using the JSON editor on the translation page.
+```json
+{
+  "in-app": {
+    "title": "Welcome to Novu!",
+    "message": "Hi {{subscriber.firstName}}, let's get started!"
+  },
+  "email": {
+    "subject": "Get Started",
+    "body": "Thank you for joining us, {{subscriber.fullName}}"
+  }
+}
+```
+
+You can export and re-upload these JSON files when translation updates are needed or use the JSON editor on the translation page to update the translations.
+
+![Translation JSON format](/images/platform/workflow/translations/translations_locale_content_edit.gif)
+
+### Exporting and importing translations
+
+#### Export
+
+- Navigate to the Translations section.
+- Click **Export en_US.json** to download the latest default file.
+- Use this as a base for other translations.
+
+#### Import
+
+- Click **Import locale(s)**.
+- Upload JSON files (e.g., fr_FR.json, es_ES.json).
+- Novu detects the locale automatically.
+
+💡 Translation changes in default locale (e.g., en_US) will mark other locales as *Outdated, needs update*
+
+## Using translation keys in step content editor
+
+In Email or In-App editors, use `{{t.key}}` to insert a translatable string. If the key already exists for the locale, it appears in the suggestions list.
+
+Clicking **Add to Translations** automatically adds a new key to default locale.
+
+Example:
+
+```html
+{{t.subject}} → "Let's get you started"
```

Comment:
We also support nested keys like `{{t.title.button.submit}}`, maybe worth mentioning